CheatSheets/windows-cheatsheet.md

```diff
@@ -68,6 +69,14 @@ created: 2022-10-20
 |` Win + Tab`| Open task view|
 |` Win + M`| Minimize all windows |
 |` Win + R`| Open the Run dialog box |
+|` Win + A`| Open the Action Center |
+|` Win + C`| Launch Cortona in listening mode (used for voice commands) |
+|` Win + S`| Search/Launch Cortona (listening mode) |
+|` Win + I`| Open the settings app |
```

Comment:
```suggestion
```
This command already exists in the `Setting Shortcuts` section. Please, delete your newly added `Win + I` command.